examples/plot_6_function_identifier.py

```diff
@@ -49,7 +49,7 @@
              size=fontsize,
              transform=fig.transFigure,
              bbox=dict(boxstyle='larrow', fc="w", ec="k"))
-#
+
 plt.draw()
 # To avoid matplotlib text output
 plt.show()
```

Comment:
again, this shouldn't be needed?  Also the rendered output shows all the text boxes overlapping, might as well reduce the `fontsize` variable while we're overhauling things.

Reply:
Good point - reduced font size and padding of the arrows.